packages/acpica.rb

```diff
@@ -2,11 +2,11 @@
 
 class Acpica < Package
   description 'ACPI tools, including Intel ACPI Source Language compiler'
-  homepage 'https://acpica.org/'
+  homepage 'https://www.intel.com/content/www/us/en/developer/topic-technology/open/acpica/overview.html'
   version '20200925'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://acpica.org/sites/acpica/files/acpica-unix-20200925.tar.gz'
+  source_url 'https://downloadmirror.intel.com/774848/acpica-unix-20191213.tar.gz'
   source_sha256 'd44388e21e3d2e47c6d39e9c897935d3f775f04fec76271dcba072c74f834589'
   binary_compression 'tar.xz'
```

Comment:
Should we not rebuild these packages of we are updating the sources? And also change the `binary_compression` field then to `tar.zst`?

Reply:
Also, this replacement source appears to be older?

Reply:
Oops, I copied the wrong url-- will fix when I wake up.

Reply:
https://github.com/acpica/acpica/files/14718333/acpica-unix-20240321.tar.gz ?